libs/remix-lib/src/execution/txRunnerWeb3.ts

```diff
@@ -227,13 +231,26 @@ export class TxRunnerWeb3 {
     })
   }
 
-  async sendUserOp (tx) {
+  async sendUserOp (tx, chainId) {
+    // AA03: Add network name and public URL to support contract transactions using smart account
+    const aaSupportedNetworks = {
```

Comment:
these constants should be put in an helper, so you don't duplicate it.

Reply:
done